support-frontend/app/wiring/Controllers.scala

```diff
@@ -201,7 +200,7 @@ trait Controllers {
     actionRefiners,
     appConfig.guardianDomain,
     appConfig.identity.webappUrl,
-    () => AwsCloudWatchMetricPut(client)(setupWarningRequest(appConfig.stage))
+    () => AwsCloudWatchMetricPut(AwsCloudWatchMetricPut.client)(setupWarningRequest(appConfig.stage))
```

Comment:
there's something weird about how you have to call AwsCloudWatchMetricPut.  I didn't bother to work out what it should look like in this PR as it is consistent, so I didn't have to pick a "winning" API like I did with S3.